portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestDXPSide.testcase

```diff
@@ -18,8 +18,6 @@ definition {
 		}
 
 		task ("Set up instance and sign in DXP") {
-			TestCase.setUpPortalInstance();
-
 			ACUtils.launchDXP();
 		}
```

Comment:
Hey @marcosapmf, you need to revert this

Reply:
ops :P